### PR TITLE
Bug fixes

### PR DESCRIPTION
**Pull request by alena1108**
1) Host networking + Alias service fixes

https://github.com/rancher/rancher/issues/1792
https://github.com/rancher/rancher/issues/1793

2) Account purge 

Cleanup only non-service instances as service instances get removed during environment.remove process triggered earlier

### CODE REVIEW

File: code/iaas/model/src/main/java/io/cattle/platform/core/dao/InstanceDao.java [hunk 1]
```diff
@@ -19,4 +20,6 @@
      */
     List<? extends Service> findServicesFor(Instance instance);
 
+    List<? extends Instance> listNonRemovedInstances(Account account, boolean forService);
```
Comment (sonchang) [inline code/iaas/model/src/main/java/io/cattle/platform/core/dao/InstanceDao.java]: Is there somewhere else that makes this call passing in forService=true? (subsequent PR?)

**Comment by sonchang** [comment]
LGTM

File: code/iaas/model/src/main/java/io/cattle/platform/core/dao/InstanceDao.java (hunk 1 above)
Comment (alena1108) [inline code/iaas/model/src/main/java/io/cattle/platform/core/dao/InstanceDao.java]: @sonchang No subsequent PR, I've just wanted this method to handle both cases - return non-service and service instances.

**Comment by cjellick** [comment]
LGTM

**Pull request merged**